### DISCUSSION TITLE
How Would You Configure this Digit

### DISCUSSION THREAD

**Discussion by jtricerolph** [jomjol/AI-on-the-edge-device#1774] [General] [answered]
Any sugestions on how to configure the ROI for the decimal digit on this meter?
Also is there a way to tweek the inner diget box vs the area box as i have had to include space outside the diget window/home to get the digit itself fitting the inner marks corectly.
![image](https://user-images.githubusercontent.com/107147602/210845373-c090b773-7e6e-4860-b77a-9db133f3b657.png)
large decimal dial digit
![image](https://user-images.githubusercontent.com/107147602/210845444-febfcbc2-08c6-460d-810f-507e01338286.png)

**Comment by jomjol** [comment, marked as answer]
The dec1 will be hard up to impossible. There is no way of tweeking the boxes, because they are only guidlines for the image and represent the proportions of the images within the training data.

**Reply by jtricerolph** [reply]
Thought as much, will stick to whole kwh I guess as was causing issues

**Comment by ghost** [comment]
> Any sugestions on how to configure the ROI for the decimal digit on this meter?
> Also is there a way to tweek the inner diget box vs the area box as i have had to include space outside the diget window/home to get the digit itself fitting the inner marks corectly.
> ![image](https://user-images.githubusercontent.com/107147602/210845373-c090b773-7e6e-4860-b77a-9db133f3b657.png)
> large decimal dial digit
> ![image](https://user-images.githubusercontent.com/107147602/210845444-febfcbc2-08c6-460d-810f-507e01338286.png)

Could you show the recognition page? I think it could work

**Reply by ghost** [reply]
A bit smaller in height, could be.

**Reply by jtricerolph** [reply]
> A bit smaller in height, could be. 

I've dropped the eecimal for now as was causing issues.
Ideally think others would be whole only and decimal continuous but not sure how to do seperate models for digital.

I've tried tweeking but can't get the size right to have the number fit inner box and window on the window border 😐
